theme/migrations/0041_remove_quotamessage_enforce_quota_and_more.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 4.2.20 on 2025-05-05 16:27
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('theme', '0040_remove_userquota_allocated_value_and_more'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='quotamessage',
+            name='enforce_quota',
+        ),
+        migrations.RemoveField(
+            model_name='quotamessage',
+            name='grace_period',
+        ),
+        migrations.RemoveField(
+            model_name='quotamessage',
+            name='grace_period_content_prepend',
+        ),
+        migrations.RemoveField(
+            model_name='quotamessage',
+            name='hard_limit_percent',
+        ),
+        migrations.RemoveField(
+            model_name='userquota',
+            name='grace_period_ends',
+        ),
```

Comment:
need to remove `grace_period_ends` here
https://github.com/hydroshare/hydroshare/blob/5680-minio-quota/hs_core/templates/email/act_on_quota_request.html#L16
as well as other places